CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to Jenkinsfile Runner
+
+This page provides information about contributing code to the Jenkinsfile Runner codebase.
+
+## Getting started
+
+1. Fork the repository on GitHub
+2. Clone the forked repository to your machine
+3. Install the development tools. In order to develop Jenkinsfile Runner, you need the following tools:
+  * Java Development Kit (JDK) 8.
+     - We usually use [OpenJDK](http://openjdk.java.net/) but you can use other JDKs as well.
+  * Maven 3.5.3 or above.
+  * Any IDE which supports importing Maven projects.
+  * Docker.
+  * Make.
+
+## Building and Debugging
+
+The build flow for Jenkinsfile Runner is built around Maven and Docker.
+
+Run `mvn clean package` for the project to be just built and `mvn clean package -Denvironment=test` for the project
+to be built and the integration tests, based on the [Jenkinsfile Runner Test Framework](https://github.com/jenkinsci/jenkinsfile-runner-test-framework), to be launched.
```

Comment:
The commas and pauses make the reading difficult. You might loose the idea of being launching the tests. What about something like this?
```suggestion
to be built and the integration tests to be launched, which are based on the [Jenkinsfile Runner Test Framework](https://github.com/jenkinsci/jenkinsfile-runner-test-framework).
```